airbyte-integrations/connectors/source-mailchimp/source_mailchimp/manifest.yaml

```diff
@@ -0,0 +1,366 @@
+      data_field: "categories"
+
+  interests_stream:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/retriever"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/interest_categories_stream"
+            parent_key: id
+            partition_field: id
+    $parameters:
+      name: "interests"
+      primary_key: "id"
+      path: "lists/{{ stream_slice.parent_slice.id }}/interest-categories/{{ stream_slice.id }}/interests"
+      data_field: "interests"
+
+  reports_stream:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_incremental_stream"
+    $parameters:
+      name: "reports"
+      primary_key: "id"
+      path: "reports"
+      data_field: "reports"
+      cursor_field: "send_time"
+
+  segments_stream:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_incremental_stream"
+    retriever:
+      $ref: "#/definitions/retriever"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/lists_stream"
+            parent_key: id
+            partition_field: id
+    $parameters:
+      name: "segments"
+      primary_key: "id"
+      path: "/lists/{{ stream_slice.id }}/segments"
+      data_field: "segments"
+      cursor_field: "updated_at"
+
+  segment_members_stream:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/retriever"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/segments_stream"
+            parent_key: id
+            partition_field: id
+      record_selector:
+        $ref: "#/definitions/retriever/record_selector"
+        record_filter:
+          type: CustomRecordFilter
+          class_name: source_mailchimp.components.MailChimpRecordFilter
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%S%z"
+      datetime_format: "%Y-%m-%dT%H:%M:%S%z"
+      cursor_field: "{{ parameters['cursor_field'] }}"
+      start_datetime:
+        type: MinMaxDatetime
+        # TODO: ref to use datetime as optional parameter if possible
+        datetime: "{{ config.get('start_date', day_delta(-500, '%Y-%m-%dT%H:%M:%S.%fZ')) }}"
+        datetime_format: "%Y-%m-%dT%H:%M:%S.%fZ"
+      step: "P100Y"
+      cursor_granularity: "PT0.1S"
+    transformations:
+      - type: AddFields
+        fields:
+          - path: ["segment_id"]
+            value: "{{ stream_slice.id }}"
+      - "#/definitions/transformer_remove_empty_fields"
+    $parameters:
+      name: "segment_members"
+      primary_key: "id"
+      path: "/lists/{{ stream_slice.parent_slice.id }}/segments/{{ stream_slice.id }}/members"
+      data_field: "members"
+      cursor_field: "last_changed"
+
+  unsubscribes_stream:
```

Comment:
Differences observed:
* Empty strings were removed from from `merge_fields`
* The parent stream `lists_stream` was not sliced in the previous implementation
* `segments` was not sliced in the previous implementation

Reply:
updated, slicing removed